apps/eclipse/src/app/global.css

```
@@ -5,26 +5,26 @@
 @import "@prisma-docs/ui/styles";
 
 #nd-sidebar {
-    @apply border-e;
-    background-color: var(--color-background);
+  @apply border-e;
+  background-color: var(--color-background);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)